app/models/jsonld_record.rb

```diff
@@ -92,7 +93,8 @@ def title_language
   def metadata_map
     {
       author_display:        'creator',
-      description_display:   'description',
+      call_number_browse_s:  'call_number',
```

Comment:
@tampakis Do you know why there are sometimes multiple identical call numbers in `call_number_display`?  e.g., https://bibdata.princeton.edu/bibliographic/1234567/solr